### PR TITLE
Fix Toastr error display

### PR DESCRIPTION
**Pull request by ffflorian**
Today I encountered an SQL error and additionally the error couldn't be displayed:
`Uncaught SyntaxError: missing ) after argument list     (index):128`.

The culprit was found quickly:
```html
<script>
toastr["error"](`SQLSTATE[42S22]: Column not found: 1054 Unknown `long_url_hash` in `where clause` (SQL: select * from `links` where `long_url_hash` = 3334713191 and `long_url` = [redacted] and `is_custom` = 0 and `secret_key` =  limit 1)`, "Error")
</script>
```
JavaScript interprets the SQL table and column names as end of the statement.

I'm not sure about the SQL error but with this small change the error is at least visible in Toastr.

### CODE REVIEW

**Comment by cydrobolt** [comment]
Using double quotes would allow only for a single-line string to be displayed. Are we sure that all errors displayed will be in a single line only? Using the backtick symbol allows multi-line strings to be passed to Toastr.

**Comment by cydrobolt** [comment]
In regards to the SQL error, it seems like you upgraded Polr without migrating the database or updating composer, as highlighted in the upgrade guide. Migrating your database should fix the SQL error. 
http://docs.polr.me/en/latest/user-guide/upgrading/

**Comment by ffflorian** [comment]
@cydrobolt 
Yes, you are right about the multi-line strings. What do you think about my updated solution?

Regarding the SQL error: Yes, I forgot the upgrading. 🙁

File: resources/views/layouts/base.blade.php [hunk 1]
```diff
@@ -60,22 +60,22 @@
...
     @if (Session::has('success'))
-        toastr["success"](`{{session('success')}}`, "Success")
+        toastr["success"](`{{str_replace('`', '&#96;', session('success'))}}`, "Success")
     @endif
 
     @if (count($errors) > 0)
         // Handle Lumen validation errors
         @foreach ($errors->all() as $error)
-            toastr["error"](`{{$error}}`, "Error")
+            toastr["error"](`{{str_replace('\`', $error)}}`, "Error")
```
Comment (cydrobolt) [inline resources/views/layouts/base.blade.php]: Why does this one have a `\` but the other ones don't?

**Comment by ffflorian** [comment]
Because it's a regex. We replace all `\`` in the error messages.

**Comment by cydrobolt** [comment]
Don't the other `str_replace` instances also use regex?

**Comment by ffflorian** [comment]
Ah yes, you are right - my mistake. Will update tomorrow.

**Comment by cydrobolt** [comment]
I fixed the escaping logic by using `` \` `` instead of using the HTML entity to escape the backticks. I've merged the PR. 

Thank you for your contributions!

**Comment by ffflorian** [comment]
Hey, sorry I totally forgot about this. Thanks for merging and fixing :)

**Pull request merged**